components/x-follow-button/src/styles/main.scss

```diff
@@ -0,0 +1,10 @@
+$o-icons-is-silent: false;
+$o-colors-is-silent: false;
+$o-typography-is-silent: false;
```

Comment:
no non-silent origami modules please. it doesn't look like the classes from these are actually used so i think you can just remove these lines.